api/shape/to.js

```diff
@@ -0,0 +1,31 @@
+import Group from './Group.js';
+import Shape from './Shape.js';
+import { getLeafs } from '@jsxcad/geometry';
+import { invertTransform } from '@jsxcad/algorithm-cgal';
+
+export const to =
+  (selection, ...ops) =>
+  (shape) => {
+    if (ops.length === 0) {
+      ops.push(() => shape);
+    }
+    ops = ops.map((op) => (op instanceof Function ? op : () => op));
+    // We've already selected the item for reference, e.g., s.on(g('plate'), ...);
```

Comment:
These inline comments are really nice for understanding what's going on

Reply:
Good :)